File: iniciante/1020/1020.c
// 1020 - Idade em Dias - beecrowd
// Iniciante - Nível 2

#include <stdio.h>

int main() {

    int idade_dias;
    scanf("%d", &idade_dias);

    int anos = idade_dias / 365;
    idade_dias %= 365;

    int meses = idade_dias / 30;
    idade_dias %= 30;

    int dias = idade_dias;

    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", anos, meses, dias);

    return 0;
}

File: iniciante/1020/1020.py
# 1020 - Idade em Dias - beecrowd
# Iniciante - Nível 2

idade_dias = int(input())

anos = idade_dias // 365
idade_dias %= 365

meses = idade_dias // 30
idade_dias %= 30

dias = idade_dias

print(anos, " ano(s)\n", meses, " mes(es)\n", dias, " dia(s)", sep="")
